content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -1,268 +1,349 @@
+
+   .. image:: lots/new-lot-number.png
+      :align: center
+      :alt: New lot number creation form with assigned product.
 
-The :guilabel:`Traceability Report` can also include a :guilabel:`Reference` receipt from the
-previous purchase order, if the product quantities shared the same lot number.
+After a new lot number has been created, saved, and assigned to the desired product, the lot number
+is saved as an existing lot number linked to the product, and can be selected when :ref:`assigning
+lot numbers to products on a receipt <inventory/product_management/assign-lots>`, or when making an
+inventory adjustment.
+
+.. example::
+   After creating the lot number, `000001` appears as an option for `Drawer Black` when assigning
+   lot numbers on the :guilabel:`Inventory Adjustment` page.
+
+   .. image:: lots/inventory-adjustment.png
+      :align: center
+      :alt: Show how to assign lot numbers on the Inventory Adjustment page.
 
 Manage lots for different operations types
 ==========================================
 
-In Odoo, the creation of new lots is only allowed upon **receiving** products from a purchase order,
-by default. **Existing** lot numbers cannot be used.
-
-For sales orders, the opposite is true: new lot numbers cannot be created on the delivery order,
-only existing lot numbers can be used.
+By default, new lots can only be created when receiving products, and existing lot numbers cannot
+be used. For sales orders, only existing lot numbers can be utilized, and new ones cannot be created
+on the delivery order.
 
 To change the ability to use new (or existing) lot numbers on any operation type, go to the
 :menuselection:`Inventory app --> Configuration --> Operations Types`, and select the desired
-:guilabel:`Operation Type`.
-
-For :guilabel:`Receipts`, found on the :menuselection:`Operations Types` page, the :guilabel:`Use
-Existing Lots/Serial Numbers` option can be enabled, by clicking :guilabel:`Edit`, and then clicking
-the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes.
+operation type.
 
-For :guilabel:`Delivery Orders`, the :guilabel:`Create New Lots/Serial Numbers` option can be
-enabled, by clicking :guilabel:`Edit`, and clicking the checkbox beside the :guilabel:`Create New
-Lots/Serial Numbers` option. Be sure to click the :guilabel:`Save` button to save all changes.
+On the operation type form, under the :guilabel:`Lots/Serial Numbers` section, tick the
+:guilabel:`Create New` checkbox to enable new lot numbers to be created during this operation type.
+Choose :guilabel:`Use Existing ones` if only existing lot numbers can be selected.
 
-.. image:: lots/lots-operations-type-form.png
+.. image:: lots/operation-type-form.png
    :align: center
    :alt: Enabled traceability setting on operations type form.
```

Comment:
This image is a little different from runbot:
![2024-05-06_12-10](https://github.com/odoo/documentation/assets/91896312/53260a7d-3a1c-48f3-9831-47ee71a74f6c)

Reply:
Omg right, the font looks different somehow! I feel super confused. We're both in 17.0 runbot, but why does it look different?
![image](https://github.com/odoo/documentation/assets/23611711/d2d719e5-ee19-441d-a13c-675c551204d3)